### ISSUE TITLE
Missing translation [Italian]

### ISSUE THREAD

**Issue by igor-cali** [pulb/mailnag-gnome-shell#56]
Which file shall I modify to include translations (Italian in my case)?
I would like to add translation and propose a pull request.
Thanks in advance.

**Comment by vantu5z** [comment]
Yes, I would like to add Russian translation. How can we do this?

**Comment by pulb** [comment]
Unfortunately there is no gettext translation in place yet (like in the main mailnag project). Contributions are welcome :)

**Comment by vantu5z** [comment]
Add PR #72 with localization support.
